README.md

```diff
@@ -4,4 +4,4 @@
 
 You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it
 
```

Comment:
Please, kindly add the Getting started headers by telling the user how to have your project running locally on their machine. Example:
### **Getting started**
To get a local copy up and running follow these simple example steps.

- How to clone the repo, how to move into the cloned repo,
- Prerequisite: e.g vscode editor, a web browser(chrome, etc).